### DISCUSSION TITLE
How can I download the PyCUDA that support specific CUDA and python version?

### DISCUSSION THREAD

**Discussion by jueqiang** [inducer/pycuda#472] [Q&A]
Link https://www.lfd.uci.edu/~gohlke/pythonlibs/#pycuda is not working and I could not find the specific version of PyCUDA that supports specific CUDA and python version (i.e. CUDA=10.1 python=3.8). How can I download the PyCUDA that support specific CUDA and python version.
![image](https://github.com/user-attachments/assets/c284a56e-12bd-4b41-9dd6-adf2ba5d9dc1)

**Comment by inducer** [comment]
We only ship source packages at this stage. If you would like binary wheel support, check out https://github.com/inducer/pyopencl for roughly equivalent functionality.
